### PR TITLE
Salary System 2.0 Handbook

### PR DESCRIPTION
**Pull request by dmurko**
Adapting to the new system.

Ref: https://github.com/niteoweb/operations/issues/1220

### CODE REVIEW

File: 5_People/career.md [hunk 1]
```diff
@@ -22,41 +24,35 @@ This is the Salary Formula:
...
-
-For the purposes of the Salary System, we take your official permanent residence (address stated in your passport), and do not change it for seasonal relocations or longer travel. If your city is not in Numbeo, or has less than 180 entries in the last 18 months, find the nearest city that qualifies.
+For the purposes of the Salary System, we take your official permanent residence (address stated in your passport), and do not change it for seasonal relocations or longer travel. 
 
 We pay Niteans relatively more in inexpensive cities, increasing their local purchasing power. We do this by compressing the Location Factor of cities below affordability ratio upwards by 2/3 of the distance between location and affordability ratio. Inversely, cities above the affordability ratio are compressed downwards by 2/3 of the distance between location and affordability.
```
Comment (dmurko) [inline 5_People/career.md]: @zupo Nothing changed here from before, so is it still correct?

File: 5_People/career-operations.md [hunk 2]
```diff
@@ -1,38 +1,44 @@
...
+## Support Manager
 
-* Deep knowledge about the project and its users by regularly talking to users either in person or by phone. 
+* Manages other tech support Niteans.
 * Designs and maintains the structure of support documentation.
-* Capable of running and directing whole teams for smaller projects.
+* Communicates with programmers and project leads about the product development.
+* Salary.com role name for base salary: `IT Project Manager I`
+
+## Operations Manager
```
Comment (zupo) [inline 5_People/career-operations.md]: The description of the roles is much closer to `Project Manager`. I.e. doing everything about a single project. Next step is doing operations on the company level, across multiple projects.

File: 5_People/career-technical.md [hunk 3]
```diff
@@ -2,31 +2,34 @@
...
 
-Since we are a software company, the base salary comes from the "Software Engineer" family of positions on Glassdoor. But we don't differentiate between different engineering careers. In other words, a data scientist is paid the same as a frontend developer and the same as a DevOps engineer.
+Since we are a software company, the base salary comes from the "Software Engineer" family of positions on Salary.com. But we don't differentiate between different engineering careers. In other words, a data scientist is paid the same as a frontend developer and the same as a DevOps engineer.
 
 The top two positions are two branches of the engineering career:
 * Lead Engineer means you start taking over managerial tasks and leading people.
 * Principal Engineer means you take ownership of all of technical nitty-gritty details in the company.
 
-## Junior Programmer / Sysadmin / Scientist 
+## Junior Engineer / Sysadmin / Scientist
```
Comment (zupo) [inline 5_People/career-technical.md]: Can remove `sysadmin / scientist`

File: 5_People/career.md [hunk 4]
```diff
@@ -22,41 +24,35 @@ This is the Salary Formula:
...
 
-### Base Salary
-
-Niteo Base Salary is calculated by first taking the [*Glassdoor salary for the role in San Francisco*](https://www.glassdoor.com/Salaries/san-francisco-software-engineer-salary-SRCH_IL.0,13_IM759_KO14,31.htm), converting it to EUR and finally discounting it by our *affordability ratio*. 
-
-#### Why San Francisco?
+#### Base Salary
 
-We take San Francisco because we assume it has the best data for the tech sector.
+Niteo Base Salary is calculated by first taking the [*Salary.com salary base for the role in United States*](https://www.salary.com/research), converting it to EUR and finally discounting it by our *affordability ratio*.
```
Comment (zupo) [inline 5_People/career.md]: Not true, we no longer discount salaries by afforability ratio. The locationFactors are compressed towards affordability ratio. See https://github.com/niteoweb/salarycalc/pull/44/files#diff-0f1be8eec416e71c28b41efb0544df26R4

File: 5_People/career.md [hunk 5]
```diff
@@ -22,41 +24,35 @@ This is the Salary Formula:
...
+To avoid day-to-day currency fluctuations we take [the 10-year exchange rate average](https://www.ofx.com/en-au/forex-news/historical-exchange-rates/yearly-average-rates/) to convert the Salary.com amounts from USD to EUR, Niteo's primary currency. The exchange rate for 2020 is _1.20_.
 
-#### Affordability Ratio
+##### Affordability Ratio
 
-We are not a VC funded Sillicon Valey startup who scales for the sake of scaling and often does not care how much things cost. We strive to be profitable so [every Nitean gets a cut](https://github.com/niteoweb/handbook/blob/master/5_People/profit-sharing.md). Hence, we discount Glassdor amounts by an *affordability ratio*, which we get by averaging Numbeo Location Ratios of Amsterdam, Barcelona, Berlin, and Lisbon. The affordability ratio for 2019 is _0.53_.
+We are not a VC funded Sillicon Valey startup who scales for the sake of scaling and often does not care how much things cost. We strive to be profitable so [every Nitean gets a cut](https://github.com/niteoweb/handbook/blob/master/5_People/profit-sharing.md). Hence, we discount Salary.com amounts by an *affordability ratio*. The affordability ratio for 2020 is _0.49_.
```
Comment (zupo) [inline 5_People/career.md]: Again not true, see https://github.com/niteoweb/salarycalc/pull/44/files#diff-0f1be8eec416e71c28b41efb0544df26R121.

File: 5_People/career.md [hunk 6]
```diff
@@ -22,41 +24,35 @@ This is the Salary Formula:
...
 
-#### How often do base salaries get calculated?
+##### How often do base salaries get calculated?
 
 Once a year, in January, during our [bi-annual IRL](https://blog.niteo.co/irls/). Same for the affordability ratio.
 
+#### Location Factor
 
-### Location Factor
+Location factor takes into account the living costs. We determine them by comparing Numbeo's [cost of living between United States and the Nitean's country](https://www.numbeo.com/cost-of-living/compare_cities.jsp?country1=United+States&city1=San+Francisco%2C+CA&country2=Germany&city2=Berlin) and calculating the *Numbeo Location Ratio*.
```
Comment (zupo) [inline 5_People/career.md]: Location Factors are compressed towards our affordability ratio: https://github.com/niteoweb/salarycalc/pull/44/files#diff-0f1be8eec416e71c28b41efb0544df26R121

Comment (zupo) [reply 5_People/career.md]: The link is comparing cities, it should compare countries.

File: 5_People/career.md [hunk 7]
```diff
@@ -22,41 +24,35 @@ This is the Salary Formula:
...
 We pay Niteans relatively more in inexpensive cities, increasing their local purchasing power. We do this by compressing the Location Factor of cities below affordability ratio upwards by 2/3 of the distance between location and affordability ratio. Inversely, cities above the affordability ratio are compressed downwards by 2/3 of the distance between location and affordability.
 
-In other words, if you live in an inexpensive city, your salary will be _decreased_ less and if you live in an expensive city, your salary will be _increased_ less by the location factor. 
-
-This way we are motivated to hire in inexpensive regions, while at the same time providing a better salary for those Niteans. All things being the same, we will always hire a Nitean from a less expensive city.
+In other words, if you live in an inexpensive country, your salary will be _decreased_ less and if you live in an expensive country, your salary will be _increased_ less by the location factor. 
 
+This way we are motivated to hire in inexpensive regions, while at the same time providing a better salary for those Niteans. All things being the same, we will always hire a Nitean from a less expensive country.
```
Comment (zupo) [inline 5_People/career.md]:
```suggestion
This way we are motivated to hire in inexpensive regions, while at the same time providing a better salary for those Niteans. All things being the same, we will always hire a candidate from a less expensive country.
```

File: 5_People/career.md (hunk 1 above)
Comment (zupo) [reply 5_People/career.md]: All good.

File: 5_People/career-operations.md [hunk 8]
```diff
@@ -26,15 +26,15 @@ You start at the tech support desk answering enquiries by understanding the user
...
 * Deep knowledge about a project and its users by regularly talking to users either in person or by phone. 
 * Is responsible that everything is running smoothly on the project.
 * Takes lead on project product development.
 * Usually at least 2-3 years of project domain knowledge.
 * Salary.com role name for base salary: `IT Project Manager II`
 
-## Senior Operations Manager
+## Operations Manager
 
 * Basically a Chief Operations Office.
```
Comment (zupo) [inline 5_People/career-operations.md]: Officer.

File: 5_People/career-operations.md [hunk 9]
```diff
@@ -1,38 +1,44 @@
...
 
-* Handles basic support questions, mostly by knowing where to find answers in our docs.
-* Usually less than 2 years of experience being in Customer Support.
-
-## Customer Support Associate
-
-* Write and improves support documentation.
-* Handles billing issues.
+* Handles a large range of support questions, many of them with technical issues to resolve.
+* Writes and improves support documentation.
```
Comment (marbeann) [inline 5_People/career-operations.md]: Maybe we can remove `support` to make it more general and less constricting. Documentations are being moved to the devops repo. And since we are now hiring technical people, they might also need to update other related docs.

File: 5_People/career.md [hunk 10]
```diff
@@ -1,4 +1,6 @@
-# Niteo Salary System
+# Career in Niteo
```
Comment (marbeann) [inline 5_People/career.md]: Maybe Niteo Career? Career in Niteo sounds off for me. No strong opinion though.

Usually the prep _in_ is used when referring to a field (e.g. career in Science), and then we use the prep _at_ when it alludes to a specific location (works at Google). Again, no strong opinion.

**Comment by zupo** [comment]
@dmurko: you merged too soon. CircleCI is reporting two broken links: https://app.circleci.com/jobs/github/niteoweb/handbook/29

```
> Links 
  1. [L13] 404 https://github.com/niteoweb/handbook/blob/master/5_People/salary-system.md#severance-policy  
  2. [L13] 404 https://github.com/niteoweb/handbook/blob/master/5_People/salary-system.md#working-hours  
```
